TACC-Cloud/hazmapper
# Task/DES-1988: Save to project metadata

## Overview: ##
Implements file save in client and adds post to project metadata on saving a map.
Requires #69 to be merged. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-1988](https://jira.tacc.utexas.edu/browse/DES-1988)

## Summary of Changes: ##

## Testing Steps: ##
1. Test with tacc-cloud/geoapi#68 to ensure that saving works through only the client.
2. Ensure that the metadata is updated for DesignSafe.

## UI Photos:

## Notes: ##
**TODO**
* [ ] Merge #69
